frontend/www/js/user.edit.js

```diff
@@ -17,7 +17,12 @@ omegaup.OmegaUp.on('ready', function() {
         $('#state_id option').remove();
         $('#state_id').val('');
 
-        var country = iso3166.country($('#country_id').val());
+        try {
```

Comment:
you can use [short-circuit evaliation](https://en.wikipedia.org/wiki/Short-circuit_evaluation) to avoid try/catch.

```
var country = iso3166.country($('#country_id').val()) || '';
```

Reply:
var country = iso3166.country($('#country_id').val()) || ''; 
Not working :(

Reply:
please elaborate. "not working" does not allow you yo help resolve the issue.

Reply:
Still giving the same error as before  TypeError: Cannot read property 'trim' of null
    at Object.country (iso3166.min.js?ver=2c7e40:9)

Reply:
oh that might be because the short-circuit evaluation should be within the call to `.country()`:

```suggestion
        var country = iso3166.country($('#country_id').val() || '');
```